kanidm_proto/src/v1.rs

```diff
@@ -463,6 +463,17 @@ impl PartialEq for AuthMech {
     }
 }
 
+impl fmt::Display for AuthMech {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            AuthMech::Anonymous => write!(f, "Anonymous (no credentials)"),
```

Comment:
Not sure if this makes sense, but if you want to make things easy for multi-lingual support, you might want to convert these to indexable codes that a client/UI can map and render themselves.

Reply:
Actually given that this is CLI support, this probably does not matter.

Reply:
It's a good idea to think about multi-lingual yes, but as you say, I think that's going to be more the focus of the WebUI more than anything.